From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Mon, 12 Jan 2015 14:14:42 +0100
Subject: [PATCH] HACK: sd-pppoe: don't use linux/ppp-ioctl.h

This is required for glibc <= 2.13

Upstream requires glibc 2.14 but this is useful for host-systemd.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 src/libsystemd-network/sd-pppoe.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/libsystemd-network/sd-pppoe.c b/src/libsystemd-network/sd-pppoe.c
index cd5a204f8c61..59c6b0917490 100644
--- a/src/libsystemd-network/sd-pppoe.c
+++ b/src/libsystemd-network/sd-pppoe.c
@@ -23,7 +23,6 @@
 
 #include <sys/ioctl.h>
 #include <linux/ppp_defs.h>
-#include <linux/ppp-ioctl.h>
 #include <net/if.h>
 #include <netinet/in.h>
 #include <linux/if_pppox.h>
@@ -38,6 +37,8 @@
 #include "async.h"
 #include "utf8.h"
 
+#define PPPIOCGCHAN	_IOR('t', 55, int)	/* get ppp channel number */
+
 #define PPPOE_MAX_PACKET_SIZE 1484
 #define PPPOE_MAX_PADR_RESEND 16
 
